free-3.md

```diff
@@ -9,42 +9,94 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
-
+My idea is to create a repository visualizer DSL. It is very difficult to jump
+into a large repository of code and understand how the code works. Especially for
+people new to open source, going into GitHub and making sense of a large project
+can be very intimidating. I imagine this stops a lot of people from even beginning
+to scratch the surface in understanding how an interesting project works. If I
+could help, the experience of jumping into a new code base would be much less
+intimidating. While READMEs do a good job of explaining the project as a whole,
+there is often a lack of description of how the codebase is structured and what
```

Comment:
You'll find that built-in MVC design doesn't necessarily lend itself to good organization. A well-structured and organized codebase is helped by a language that encourages it, but it requires buy-in by developers.